packages/ia-wayback-search/src/ia-wayback-search.js

```diff
@@ -0,0 +1,78 @@
+import { html, LitElement } from 'lit-element';
+import waybackSearchCSS from './styles/wayback-search';
+import searchIcon from './icon-search';
+
+class WaybackSearch extends LitElement {
+  static get styles() {
+    return waybackSearchCSS;
+  }
+
+  static get properties() {
+    return {
+      locationHandler: { type: Object },
```

Comment:
_Suggestion_ Since the purpose of this handler in the context of this component is to handle the query execution and not necessarily the location, I might call it a `queryHandler` and when you're calling it, use something like `this.queryHandler.performQuery(query)`. `submitCallback()` doesn't really convey the action that is happening.